### ISSUE TITLE
Improve `phel fmt _DIR_`

### ISSUE THREAD

**Issue by JesusValera** [phel-lang/phel-lang#198]
Currently, when we run `php phel` we get the following output:
```
Usage: phel [command]

Commands:
    repl
        Start a Repl.

    run <filename-or-namespace>
        Runs a script.

    test <filename> <filename> ...
        Tests the given files. If no filenames are provided all tests in the
        test directory are executed.

    fmt <filename>
        Formats the given files.

    help
        Show this help message.
```

I suggest
- `phel fmt _DIR_` -> recursively format the `.phel` files inside the specified folder
- `phel fmt` -> recursively browse all dirs from the current folder (¿excluding dirs like `vendor`?)

**Comment by jenshaase** [comment]
It should be possible to define multiple files and folders. For example
```
phel fmt src/phel tests/phel path/to/file.phel
```

**Comment by jenshaase** [comment]
Fixed with #199
